### PR TITLE
Remove useless toolbar

### PR DESCRIPTION
**Pull request by Hidoyatmz**
Remove useless toolbar

### CODE REVIEW

File: src/app/shared/constants/options/conditions.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ export const CONDITION_DISTANCE_TO_CV3: Option[] = [
   { value: 1,	name: 'Distance must be higher than ConditionValue2' },
   { value: 2,	name: 'Distance must be lower than ConditionValue2' },
   { value: 3,	name: 'distance must be equal to or higher than ConditionValue2' },
-  { value: 3,	name: 'distance must be equal to or lower than ConditionValue2' },
+  { value: 4,	name: 'distance must be equal to or lower than ConditionValue2' },
```
Comment (FrancescoBorzi) [inline src/app/shared/constants/options/conditions.ts:64]: these changes aren't related with the PR's title

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Keira3",
-  "version": "3.0.0-beta.4",
+  "version": "3.0.0-rc.0",
```
Comment (FrancescoBorzi) [inline package.json:3]: these changes aren't related with the PR's title

File: src/app/shared/constants/options/conditions.ts (hunk 1 above)
Comment (Kitzunu) [reply src/app/shared/constants/options/conditions.ts:64]: + this is fixed on Master

**Comment by FrancescoBorzi** [comment]
@Hidoyatmz are you going to complete this?

**Pull request closed without merging**